alarm_table_defs.hpp

```diff
@@ -116,7 +112,9 @@ class AlarmTableDefs
   // Generate alarm table definitions based on JSON files on the node
   bool initialize(std::string& path);
 
+  //AlarmTableDef create_def();
   // Retrieve alarm definition for specified index/severity
+  void insert_def(AlarmTableDef);
```

Comment:
You've added this declaration between the comment above and the function it's describing - it looks like the comment "Retrieve alarm definition for specified index/severity" describes this function, but it actually describes `get_definition`.

On that note, it would be good to have a one-line comment explaining what this function does (inserts an AlarmTableDef into the _key_to_def list, so that it can later be retrieved with get_definition).